src/types/metaFactory.ts

```diff
@@ -64,5 +69,40 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
         }
       }
     })
+  },
+  soundCreatorV1: {
+    newContractCreatedEvent: 'SoundEditionCreated',
+    metadataAPI: async (events, clients: Clients) => {
+      const soundOfficialContracts = await clients.sound.fetchContractAddresses();
+      const editionAddresses = new Set(events.map(event => formatAddress(event.args!.soundEdition)));
+      const officialEditions = new Set([...editionAddresses].filter((address) => soundOfficialContracts.has(address)));
+      return officialEditions;
+    },
```

Comment:
todo - may want some better error handling on this api so if it's down, it doesn't block spindexer